include/st_string_priv.h

```diff
@@ -95,13 +95,13 @@ namespace _ST_PRIVATE
     }
 
     ST_NODISCARD
-    inline const char *find_cs(const char *haystack, size_t size, char ch)
+    inline const char *find_cs(const char *haystack ST_LIFETIME_BOUND, size_t size, char ch)
```

Comment:
Just checking, these annotations are intentionally on the parameter and not on the method?

Reply:
Yes, the return parameter is a reference to the same block of memory as the `haystack` variable.